src-tauri/src/platform_specific/linux.rs

```diff
@@ -24,11 +24,11 @@ pub fn linux_checks_librs() -> Result<(), String> {
     Ok(())
 }
 
-fn get_proton_dir() -> Option<String> {
-    let steam_dir = steamlocate::SteamDir::locate()?;
-    let compat_dir = format!("{}/compatibilitytools.d/", steam_dir.path.display());
+fn get_proton_dir() -> String {
+    let steam_dir = steamlocate::SteamDir::locate().unwrap();
```

Comment:
Why `.unwrap()`?

Wouldn't it be better to handle the error? ^^

Reply:
Oh nevermind, we were unwrapping before already, just one level higher lol

Reply:
what you said, plus I don't know how to handle this any better?
except maybe use an `.expect()`

Reply:
I guess the best way would be to bubble up the error to the UI so that we can handle it gracefully instead of crashing the thread.

That might be outside the scope of the PR so for now I'd just stick with returning an `Option<T>` or `Result<T, E>` like before and unwrapping higher up. That way we can push the unwraps up the stack until we finally reach the function called from UI and just return an Error type there to show an error message in UI ^^

Reply:
Reworked it all so that `get_proton_dir` now returns a Result<String, String> and that everything that is invoked externally gets String contents.